src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -1014,13 +941,11 @@ public void setDateTime(String date, String time, TimeZone tz) {
         setDateTime(dateObject);
     }
 
+    /**
+     * Currently only one itinerary is returned for a direct street search
+     */
```

Comment:
This seems to be a hack. The comment is a step in the right direction, but a accessor method should follow the accessor contract. Maybe we can inline this method, or rename it with a "long-ugly" name which clearly indicate the intention.  This is a accessor method on a public field - witch also does not make sense. Can we delete this method and instead insert 1 where it is used?

Reply:
I agree with your comments, but I would prefer to make these changes in a new PR, since following this through leads to even more cleanup of the a star class.